tasks/pcs-cib-constraint-location.yml

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure location constraint '{{
+      constraint.id | default(constraint_index)
+    }}'
+  command:
+    cmd: >
+      pcs -f {{ __ha_cluster_tempfile_cib_xml.path | quote }}
+      -- constraint location
+      {% if constraint.rule | default() %}
+        {% if constraint.resource.pattern | default() %}
+          regexp%{{ constraint.resource.pattern | quote }}
+        {% else %}
+          {{ constraint.resource.id | quote }}
+        {% endif %}
+        rule
+        {% if constraint.id | default() %}
+          constraint-id={{ constraint.id | quote }}
+        {% endif %}
+        {% if constraint.resource.role | default() %}
+          role={{
+            __ha_cluster_pcs_cli_role[constraint.resource.role | lower]
+            | default(constraint.resource.role)
+            | lower | quote
+          }}
+        {% endif %}
+        {% for option in constraint.options | default([]) %}
+          {{ option.name | quote }}={{ option.value | quote }}
+        {% endfor %}
+        {{ constraint.rule }}
+      {% elif
+        (constraint.id | default())
+        or (
+          constraint.options | default([])
+          | rejectattr('name', 'eq', 'score') | list
```

Comment:
Can't use `eq` with jinja 2.7 - use this instead: `rejectaddr('name', 'match', '^score$')`

Reply:
Fixed, thanks for the hint!